### PR TITLE
Disable Visual Bell by default

### PR DESCRIPTION
**Pull request by vramana**

### CODE REVIEW

File: dotVimRc [hunk 1]
```diff
@@ -89,7 +89,12 @@ set wildignore+=*/node_modules/**
 " Including this messes up fugitive in git mergetool:
 " set wildignore+=*/.git/*,*/.hg/*,*/.svn/*        " Linux/MacOSX
 set sm!  " show matching brace/paren
+
+" Disable visualbell by default
+" http://unix.stackexchange.com/a/5313
```
Comment (jordwalke) [inline dotVimRc]: One commend says it must also be set in .gvimrc. Did you find that to be correct?

**Comment by jordwalke** [comment]
Wow, this diff gets rid of a longstanding issue I had with using my vim setup in the command line Vim (even on Mac). It used to be that when hitting `j` towards the end of a document, once it hit the end would backlog the screen for a very long time and cause pauses for two seconds. This fixes it!

File: dotVimRc (hunk 1 above)
Comment (vramana) [inline dotVimRc]: It was working with this. But I can send another patch to disable it in gvimrc.

Comment (jordwalke) [inline dotVimRc]: *shrug* works fine for me, so it should be okay.

Comment (vramana) [inline dotVimRc]: Let's just put it anyway. It's just one line patch.

**Pull request merged**